husarion_ugv_controller/launch/controller.launch.py

```
@@ -76,6 +76,19 @@ def generate_launch_description():
         ),
     )
 
+    publish_robot_state = LaunchConfiguration("publish_robot_state")
+    declare_publish_robot_state_arg = DeclareLaunchArgument(
```

Ruff (0.15.2)

[error] Local variable `declare_publish_robot_state_arg` is assigned to but never used

Remove assignment to unused variable `declare_publish_robot_state_arg`

(F841)